etl/steps/data/garden/health/2024-04-02/organ_donation_and_transplantation.meta.yml

```diff
@@ -0,0 +1,64 @@
+definitions:
+  common:
+    presentation:
+      attribution_short: GODT
+      topic_tags:
+        - Global Health
+    processing_level: minor
+    unit: ""
+    short_unit: ""
+
+
+dataset:
+  update_period_days: 365
+
+
+tables:
+  organ_donation_and_transplantation:
+    variables:
+      n_organ_donors:
+        title: Total number of actual deceased organ donors
```

Comment:
What does 'actual deceased' mean here? Maybe something for `description_short`

Reply:
Actual means donors from whom organs are taken, as opposed to potential donors.